app/views/school_groups/map.html.erb

```diff
@@ -1,31 +1,113 @@
-<%= render 'enhanced_header' %>
+<% if Flipper.enabled?(:group_dashboards_2025, current_user) %>
+  <%= content_for :page_title, "#{@school_group.name} #{t('school_groups.titles.map')}" %>
 
-<div class="row">
-  <div class="col-md-12">
-    <div class="school-group-map map" id="geo-json-map" data-school-group-id="<%= @school_group.id %>"></div>
+  <div class="row">
+    <div class="col-12 col-md-6">
+
+      <div class="row">
+        <div class="col-8 mt-2 p-4">
```

Comment:
slight preference for without col-8 as wrap seems unnecessary

Reply:
<img width="739" height="299" alt="image" src="https://github.com/user-attachments/assets/5aff1151-563d-4796-aba6-81c3e4185848" />